bounce.py

```
@@ -23,21 +22,87 @@
+
+    extracted = tldextract.extract(host)
+    domain = f"{extracted.domain}.{extracted.suffix}"
+
+    if not validators.domain(domain):
+        return ""
+
+    return domain
+
 
 @app.route('/')
 def bounce():
```

Ruff (0.7.0)

Missing return type annotation for public function `bounce`

(ANN201)